pepper-apis/config/squid.conf

```diff
@@ -0,0 +1,22 @@
+# turn off caching
+cache deny all
+cache_dir null /tmp
+
```

Comment:
This is the set of configs that worked for me. We can probably fine-tune this more.